browser/index.js

```diff
@@ -250,14 +250,16 @@ document.addEventListener("DOMContentLoaded", function() {
         logoutMerchant();
     };
 
-    const monetizationCheck = (productIdGetter, resultContainer, fn) => async (e) => {
+    const monetizationCheck = (productIdGetter, resultContainer, fn, userId) => async (e) => {
         e.preventDefault();
         const productId = productIdGetter();
-        const result = await monetization[fn](productId);
+        const result = userId && fn === 'hasAccess'
```

Comment:
Not sure if it is good idea to reuse that function. Especially when I look at that condition...